### DISCUSSION TITLE
[Node Operator Question] Syncing node taking too long

### DISCUSSION THREAD

**Discussion by mateumiralles** [ethereum-optimism/developers#364] [Node Operators] [answered]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

The Dappnode dev team noticed that running an OP node in a dappnode with the upstream version up-to-date is taking too long to sync. After 3 days running the node, is still syncing blocks from 7 months ago. We are unsure about what else we need to do.

### Protocol Description

- op-geth version: `v1.101311.0`
- op-node version: `v1.7.3`
- network name: mainnet

### Node Logs

The logs seem fine.

- op-geth logs:
```
INFO [04-24|07:18:03.410] Imported new potential chain segment     number=110,071,673 hash=bd40e2..54ced2 blocks=1 txs=1   mgas=0.047  elapsed=2.162ms      mgasps=21.690   age=7mo15h49m snapdiffs=4.25MiB    triedirty=255.24MiB
INFO [04-24|07:18:03.411] Chain head was updated                   number=110,071,673 hash=bd40e2..54ced2 root=aa764b..14683e elapsed="72.841µs"   age=7mo15h49m
```

- op-node logs:
```
t=2024-04-24T09:36:08+0000 lvl=info msg="Sync progress" reason="processed safe block derived from L1" l2_finalized=0xf872c0a1621d1efe142ae66508747149ed89ad6f075a55c73cb0d00defa613fd:110145111 l2_safe=0x5b6e931f5a2af19146320dac45d2b31e6059c86376381efd7d868d64d6a66595:110145430 l2_pending_safe=0x5b6e931f5a2af19146320dac45d2b31e6059c86376381efd7d868d64d6a66595:110145430 l2_unsafe=0x5b6e931f5a2af19146320dac45d2b31e6059c86376381efd7d868d64d6a66595:110145430 l2_backup_unsafe=0x0000000000000000000000000000000000000000000000000000000000000000:0 l2_time=1695889637 l1_derived=0xa04fd13a85c383d2e1ce9c389dbc814698dac48e9a4f73d4fc518c5c5094da63:18233042
t=2024-04-24T09:36:08+0000 lvl=info msg="Found next batch" batch_type=SingularBatch batch_timestamp=1695889639 parent_hash=0x5b6e931f5a2af19146320dac45d2b31e6059c86376381efd7d868d64d6a66595 batch_epoch=0x2b9525bd671997d09e67082fa572cd3fad65ecb61cecccfc18d0094600cbcafd:18233034 txs=6
t=2024-04-24T09:36:08+0000 lvl=info msg="generated attributes in payload queue" txs=7 timestamp=1695889639
t=2024-04-24T09:36:08+0000 lvl=info msg="inserted block" hash=0xeb73c554cc01d5324c915ef5de5bdbe143ba875668f8887506fd3a84bbde4072 number=110145431 state_root=0x6dba161d9a648a90c006d70a3c1107be30841c3c90f84284bbe66c3e7e922719 timestamp=1695889639 parent=0x5b6e931f5a2af19146320dac45d2b31e6059c86376381efd7d868d64d6a66595 prev_randao=0xa89d3e79d8adefde4677f9ec5bd18f406cea3846c8197bb498cdd91328a53c34 fee_recipient=0x4200000000000000000000000000000000000011 txs=7 update_safe=true
```

### Additional Information

Our entrypoints.sh

- op-geth: [op-geth entrypoint](https://github.com/dappnode/DAppNodePackage-op-geth/blob/dappnodebot/bump-upstream/op-geth%40v1.101311.0/op-geth/entrypoint.sh)

- op-node: [op-node entrypoint](https://github.com/dappnode/DAppNodePackage-op-node/blob/dappnodebot/bump-upstream/optimism%40v1.7.3/op-node/entrypoint.sh)

**Comment by hash1go** [comment, marked as answer]
Hello. Snap sync is now enabled as a sync method, so please consider using it.
https://docs.optimism.io/builders/chain-operators/management/snap-sync
If sync is still slow, let us know.

I think you are using full sync.
![image](https://github.com/ethereum-optimism/developers/assets/90124956/7e0c441e-7588-4211-8860-efd9c194a951)
